examples/text-generation/utils.py

```diff
@@ -368,11 +368,46 @@ def setup_generation_config(args, model, tokenizer):
     return generation_config
 
 
+def exclude_hpu_graph_configs(args):
+    # Excluded configs for batch size 1 for hpu graph
+    if args.batch_size == 1 and args.limit_hpu_graphs:
+        if "falcon-180B" in args.model_name_or_path:
+            return False
+        if args.quant_config:
+            if (args.max_new_tokens == 32768 and args.max_input_tokens == 16384 \
```

Comment:
We will revise this change to convert the == in these conditions to >= to make them more generic

Reply:
yes, once QA confirms the test coverage, we will adjust the conditions